### PR TITLE
[BD-6222] Add strong theme to bpk-component-badge

### PR DESCRIPTION
**Pull request by giriawu**
Figma: https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=6099%3A2722

In the new design of the hotel card, we have made extensive use of `BpkBadge` which is white color on a black background.
But this theme style is not yet well supported on the web (they are already supported on iOs and Android).
So we added it to `bpk-component-badge` as the base theme - **strong**, for better consistency support and user experience.

Related PR in backpack-foundations: https://github.com/Skyscanner/backpack-foundations/pull/106

## Snapshot
image

image

Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [x] `README.md`
- [x] Tests
- [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)
Docs TK has been created.

### CODE REVIEW

**Comment by giriawu** [comment]
Will be updated bpk-mixins version after this [PR](https://github.com/Skyscanner/backpack-foundations/pull/106) is merged.

**Comment by giriawu** [comment]
With reference to this [PR](https://github.com/Skyscanner/backpack/pull/2525/files), after [the latest release of backpack-foundation](https://github.com/Skyscanner/backpack-foundations/tags), I updated the backpack component with these packages: `@skyscanner/bpk-foundations-web` , `@skyscanner/bpk-svgs`, `bpk-mixins`, `bpk-stylesheets`

**Review by jasonhuang-sky — APPROVED** [review]

File: UNRELEASED.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-**Fixed**
```
Comment (anambl) [inline UNRELEASED.md:1]: This was probably removed by accident? 😄

Comment (giriawu) [reply UNRELEASED.md:1]: I'm not sure about this, but when I contributed to the PR before, the UNRELEASED file was always empty, so I would just commit my changes. 
But this time I pulled main and found it had content in it, so I deleted it and added my own changes. Sorry about this, I will revert it back.

Comment (anambl) [reply UNRELEASED.md:1]: It's probably because some PR was merged after you started the contribution 🙂 No problem!

Comment (giriawu) [reply UNRELEASED.md]: Have updated, thanks for your review!

File: packages/bpk-component-graphic-promotion/src/BpkGraphicPromo-test.js [hunk 2]
```diff
@@ -125,12 +125,4 @@ describe('BpkGraphicPromo', () => {
 
     expect(props.onClick).toHaveBeenCalledTimes(1);
   });
-
```
Comment (anambl) [inline packages/bpk-component-graphic-promotion/src/BpkGraphicPromo-test.js:128]: Same with the graphic promo changes, they were merged in after you started the contribution 🙂

Comment (giriawu) [reply packages/bpk-component-graphic-promotion/src/BpkGraphicPromo-test.js]: Nice catch, I have updated this part. Thank you very much!

**Review by anambl — APPROVED** [review]
LGTM!

**Pull request merged**